persistence/src/tests/spi/clusterstatetest.cpp

```diff
@@ -154,35 +155,35 @@ TEST(ClusterStateTest, testReady)
     {
         lib::Distribution d(getCfg(3, 0));
         ClusterState state(s, 0, d);
-        EXPECT_EQ(false, state.shouldBeReady(b));
+        EXPECT_EQ(Trinary::False, state.shouldBeReady(b));
     }
 
     // Only node 0 with 1 ready copy.
     for (uint32_t i = 0; i < 3; ++i) {
         lib::Distribution d(getCfg(3, 1));
         ClusterState state(s, i, d);
-        EXPECT_EQ(i == 0, state.shouldBeReady(b));
+        EXPECT_EQ((i == 0) ? Trinary::True : Trinary::False, state.shouldBeReady(b));
```

Comment:
Consider adding a helper function converting from bool to trinary.

Reply:
Good idea, fixed